docs/conditions.md

```diff
@@ -1551,3 +1551,31 @@ Note: it will show or hide spells in vehicle spell bar.
 
 -   0 (If target needs to be on land)
 
+### Breadcrumb Quests
+
+Breadcrumb quests are optional quests that lead players to a quest hub or NPC but can be skipped if the player goes directly to the destination. Unlike mutually exclusive quests, a breadcrumb should only become unavailable after the player picks up or completes the main quest - NOT vice versa.
+
+**Do NOT use [ExclusiveGroup](quest_template_addon#exclusivegroup)** for breadcrumb quests. ExclusiveGroup makes quests mutually exclusive in both directions, which breaks quest progression if the player does the breadcrumb first.
+
+**Correct approach:** Use CONDITION_SOURCE_TYPE_QUEST_AVAILABLE (19) with CONDITION_QUESTTAKEN (9) and CONDITION_QUESTREWARDED (8) to hide the breadcrumb when the main quest is taken or completed.
```

Comment:
I know this is not related to the wiki, but we should be able to just handle this core side with breadcrumb quests to not need conditions for this?